Create Slack Commands guide for Datadog integration (#38248)
DataDog/documentation: content/en/integrations/guide/slack-actions.md

---

> title: Slack Commands

Reviewer: I think we ended up going with "Slack Actions" according to the [design](https://www.figma.com/design/nzMiHulvmusYPJ1CTwIrzH/Action-Tray---Slash-Commands?node-id=2012-20&t=0xz5FtfXQonmhK1y-1).

Author: Ack, made update

description: "Utilize these Slack actions for the workspaces you have set up. Follow the [Slack integration documentation for setup.][1]"

---

## Overview

> To run any Slack actions, type `/dd` in any workspace with the Datadog App installed to pull up an action tray listing all actions you can take. Alternatively, directly type out the action.

Reviewer 2: ```suggestion
Slack actions are available in any Slack workspace with the Datadog App installed. Type `/dd` in the workspace to open an action tray listing all available actions. Alternatively, type the full command directly.
```

> To run any Slack command, type /dd in any workspace with the Datadog App installed to pull up an action tray listing all actions you can take. Alternatively, directly type out the command.

Reviewer: `/dd` for readability

Author: good catch!

> ## Incidents

Reviewer 2: Is this guide supposed to be a singular reference for all commands? If so, should we be referencing out to other docs?
Is there a possibility for drift since we're documenting this in multiple places? For example

**Commands listed here that I couldn't find documented elsewhere:**
- /dd in investigate ("Trigger Bits Investigation")
- /dd shortcuts ("View Incident Actions") — the Incident Management doc documents the bare /datadog command as opening the action tray instead. Is /dd shortcuts an alias for that, or a separate command?

**Commands missing from this guide that exist elsewhere:**
- /datadog incident test (declare a test incident)
- /datadog task / /datadog task list (create/list incident tasks)

**Possible description mismatch for /dd in private:**
This guide says: "Archive the current channel, create a new private channel, and add all existing responders."
The Incident Management doc says: "Make the incident private (if private incidents are enabled)."

**Region availability:**
The Incident Management doc notes /datadog incident summary isn't available in gov/gov2 regions. Should the same caveat apply to /dd in summary here?

Reviewer: Yes! We want this guide to be the source of truth for all possible actions existing on the /dd command modal.
- /dd shortcuts is a new command that we added to invoke the incident action tray. The bare /datadog command will now invoke the command modal, so we need a new way to be able to pull up the action tray.

Will update this doc so the other pages align.

> Use the following keyboard shortcuts for incidents navigation. All commands can utilize `/dd in` as an alias for `/dd incident`. For more details, refer to [Incident documentation][2].

Reviewer 2: ```suggestion
Use the following commands for incidents navigation. All commands can use `/dd in` as an alias for `/dd incident`. For more details, see [Integrate Slack with Datadog Incident Management][2].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd in` or `/dd incident` | Declare an incident.|
| `/dd in update` or `/dd in edit`| Update the incident's title, state, severity, and attributes.|
| `/dd in responders`| Manage the incident's response team.|
| `/dd in investigate` | Trigger Bits Investigation. |
| `/dd in summary` | Generate the incident's summary with AI. |
| `/dd in notify` | Notify @-handles about the incident. |
| `/dd in list` | List open incidents.|

> | `/dd in private`| Archive the current channel, create a new private channel, and add all existing responders.|

Reviewer 2: ```suggestion
| `/dd in private`| Archive the current channel, create a private channel, and add all existing responders.|
```

| `/dd in public` | Make the incident and its timeline visible to anyone with incident-read permissions. |
| `/dd followup` | Create a new follow-up.|
| `/dd followup list`  | List Incident Follow-ups |
| `/dd shortcuts` | View Incident Actions.|

## On-Call

> Use the following keyboard shortcuts for On-Call. For more details, refer to [On-Call documentation][3].

Reviewer 2: ```suggestion
Use the following commands for On-Call. For more details, see [On-Call Pages][3].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd page` | Page an on-call team.|
| `/dd shifts`| See your upcoming on-call shifts.|
| `/dd override`| Request someone to cover an on-call shift. |

## Monitors

> Use the following keyboard shortcuts for Monitors. For more details, refer to [Alerting documentation][4].

Reviewer 2: ```suggestion
Use the following commands for Monitors. For more information on adding Slack to monitors, see [Monitor Notifications][4].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd monitors` | List monitors that are currently alerting.|

## Dashboard

> Use the following keyboard shortcuts for Dashboards. For more details, refer to [Dashboard documentation][5].

Reviewer 2: ```suggestion
Use the following commands for [Dashboards][5].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd dashboard` | Share a dashboard widget to this channel.|

## Workflows

> Use the following keyboard shortcuts for Workflows. For more details, refer to [Workflow Automation documentation][6].

Reviewer 2: ```suggestion
Use the following commands for Workflows. For more information on using Slack in Workflows, see [Trigger a workflow][6].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd workflow` | Run an automation workflow.|

## Accounts

> Use the following keyboard shortcuts for account management. For more details, refer to [Account Management documentation][7].

Reviewer 2: ```suggestion
Use the following commands for [account management][7].
```

| Command | Description|
| ------------------ | ---------- |
| `/dd accounts` | Manage your linked Datadog accounts. |

[1]: /integrations/slack/?tab=datadogforslack
[2]: /incident_response/incident_management/setup_and_configuration/integrations/slack/#slack-commands
[3]: /incident_response/on-call/pages/#through-slack
[4]: /monitors/notify/#notification-recipients

> [5]: /product_analytics/dashboards/

Reviewer 2: To confirm, should this link to Product Analytics dashboards docs or general Dashboards docs?

```suggestion
[5]: /dashboards/
```
